woocommerce-services.php

```diff
@@ -1748,6 +1764,17 @@ public function render_schema_notices() {
 				<?php
 			}
 		}
+
+		public function are_woo_shipping_and_woo_tax_active() {
```

Comment:
[Non blocking] WP standards is normally pretty strict with adding DocBlocks, but it's not a strictly followed rule on the plugin anyway.